apps/system/js/audio_channel_controller.js

```diff
@@ -235,5 +316,20 @@
     this.instanceID = this.app.instanceID + '_' + this.name;
   };
 
+  /**
+   * Send MozContentEvent to control the audio chanenl in System app.
+   *
+   * @param {Object} obj The arguments for passing to Gecko.
+   * @param {Object} obj.type The operation for the audio channel.
+   * @param {Object} obj.name The audio channel name you want to control.
+   * @param {Object} obj.volume The volume you want to set.
+   */
+  AudioChannelController.prototype._sendContentEvent = function(obj) {
+    var event = document.createEvent('CustomEvent');
```

Comment:
Sure, just follow the code[1] existed in System app.
Let's update it.

[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/activities.js#L266-L268